Xamarin.Forms.Core/AdaptiveTrigger.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+
+namespace Xamarin.Forms
+{
+	public class AdaptiveTrigger : StateTriggerBase
```

Comment:
Let's make all the triggers we are adding sealed for now

Reply:
Also same comment: Makes sense as there's nothing to override, but the UWP design doesn't have it sealed